### DISCUSSION TITLE
Decreasing disk space usage of template node

### DISCUSSION THREAD

**Discussion by serkixenos** [paritytech/substrate#14786] [Q&A]
I am experimenting with substrate template node, and I've noticed huge disk space usage when running it. There are binary files with six digit name and extension `.log` in data folder `./chains/testnet/db/full`, with the size of 2 to 5 MBs. They are generated and they are not deleted as the node runs. Node is running for 12+ hours, generated 15000 blocks so far, mostly empty, my data folder is 1.4GB in size. Now, if I restart the node, log files are deleted, and the data folder goes to like 20 MBs, and network continues to build block. For me, this means that log files are some temporary data, that is not actually needed for running or syncing the node.

My questions are: What are these files and why are they not deleted as the node runs? Is there a way to control the number or total disk space occupied by these files? I need to decrease the disk space usage as much as possible. Any advice on how to do that?

**Comment by bkchr** [comment]
Try running with `--database paritydb`. We are not building rocksdb.
